### PR TITLE
feature/20181200

### PR DESCRIPTION
**Pull request by SlavaMelanko**
1. Version has been updated to 20181200
2. Added `cxx_standard` (https://github.com/abseil/abseil-cpp/issues/218) and `build_testing` options

### CODE REVIEW

**Comment by SSE4** [comment]
link order is still important, we need to fix it to make travis happy

File: conanfile.py [hunk 1]
```diff
@@ -49,38 +55,39 @@ def package(self):
         self.copy("*.lib", dst="lib", src=".", keep_path=False)
 
     def package_info(self):
-        self.cpp_info.libs = ["absl_synchronization",                              
-                              "absl_symbolize",
-                              "absl_malloc_internal",
-                              "absl_time",
-                              "absl_strings",
+        self.cpp_info.libs = ["absl_any",
```
Comment (uilianries) [inline conanfile.py]: maybe this new order is causing Travis error. Abseil has circular dependency from itself.

Comment (SSE4) [reply conanfile.py]: definitely. the problem abseil has added some new libraries, and removed previously available, so we have to guess new order now.

Comment (SlavaMelanko) [reply conanfile.py]: Hi @SSE4 
I will try to figure out it this week

Comment (uilianries) [reply conanfile.py]: Thanks Alf! :joy_cat:

**Comment by uilianries** [comment]
@SMelanko I think you could replicate https://github.com/bincrafters/conan-abseil/blob/stable/20180600/conanfile.py#L51 in your PR

**Comment by SSE4** [comment]
@SMelanko @uilianries I've applied mentioned changes, but it still fails

**Comment by uilianries** [comment]
@SSE4 did you fix on last commit?

**Comment by SSE4** [comment]
@uilianries yep, I've found stupid bug, we'll see how it goes on CI...

**Comment by uilianries** [comment]
@SSE4 nice :+1:

**Pull request merged**